.github/workflows/MainDistributionPipeline.yml

```diff
@@ -16,15 +16,17 @@ jobs:
     name: Build extension binaries
     uses: duckdb/extension-ci-tools/.github/workflows/_extension_distribution.yml@v1.0.0
     with:
-      duckdb_version: main
+      duckdb_version: v1.0.0
```

Comment:
is this not already done? Or was this done earlier and now you are just reverting it?

Reply:
Previously we changed it with @carlopi to main and current changes are came from the @samansmink recommendation to match this file to `delta`'s along with removing the `_extension_deploy.yml`

Reply:
One thing: I think it make sense to have the duckdb submodule match what CI compiles

If sqlsmith should strive to be compatible both with v1.0.0 and main, I would have 2 CI jobs (or 2 sections of the same yml file) compile for both cases.

Reply:
> I would have 2 CI jobs (or 2 sections of the same yml file) compile for both cases.

Do you mean one for the current stable version of the submodule duckdb and one for some another version? I'm not sure I got it, how should it be changed